cloud

```diff
@@ -435,6 +435,7 @@ function pubsub_deploy() {
                           new_debug_subscription \
                           updated_topic \
                           updated_debug_subscription \
+                          notification_topic \
```

Comment:
I know the function docs, are not much use so far, but we gotta maintain consistency. Could you add the new option to the function docs?